x11-misc/trayer-srg/trayer-srg-1.1.8-r1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="trayer fork with multi monitor support and cleaned up codebase"
+HOMEPAGE="https://github.com/sargon/trayer-srg"
+SRC_URI="https://github.com/sargon/${PN}/archive/${P/-srg/}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+"
+RDEPEND="
+	x11-libs/gdk-pixbuf:2
+	x11-libs/gtk+:2
+	x11-libs/libX11
+"
+BDEPEND=""
+
+S="${WORKDIR}"/${PN}-trayer-${PV}
+
+src_configure() {
+	econf --prefix="${EPREFIX}"
```

Comment:
Are you sure this is OK? Didn't the old comment say it *wasn't* autoconf?

Reply:
Turns out that it is in fact correct, and I'm not sure why it installed without any problems on my test box.